### PR TITLE
Fix for WFGP-242, Documentation for WfInstallPlugin options

### PR DESCRIPTION
**Pull request by jfdenise**
Issue: https://issues.redhat.com/browse/WFGP-242

### CODE REVIEW

File: docs/guide/intro/index.adoc [hunk 1]
```diff
@@ -13,3 +15,129 @@ implement new WidFly subsystems and expose new server capabilities.
...
+|NONE
+|A list of Maven artifact coordinates used to override the artifact versions that are defined in the provisioned WildFly feature-pack.
+For example, to override undertow-core: `io.undertow:undertow-core:2.3.1.Final::jar`
+
+|jboss-reset-embedded-system-properties
+|Comma separated list of String
+|NONE
+|When executing a forked process to generate the server configuration, all system properties defined in the parent process are inherited by the forked process.
+In case you want to reset WildFly specific system properties, set this property to an empty value. 
+To erase additional specific system properties, set this property to a comma separated list of system property names to erase.
```
Comment (spyrkob) [inline docs/guide/intro/index.adoc:136]: does that mean that, if the system property is set via `jboss-reset-embedded-system-properties`, both Wildfly properties and the provided property will be removed?

File: docs/guide/intro/index.adoc [hunk 2]
```diff
@@ -13,3 +15,129 @@ implement new WidFly subsystems and expose new server capabilities.
...
+|'\|' separated list of maven coordinates in the following syntax: GroupId:ArtifactId:Version:[Classifier:]Extension
+|NONE
+|A list of Maven artifact coordinates used to override the artifact versions that are defined in the provisioned WildFly feature-pack.
+For example, to override undertow-core: `io.undertow:undertow-core:2.3.1.Final::jar`
+
+|jboss-reset-embedded-system-properties
+|Comma separated list of String
+|NONE
+|When executing a forked process to generate the server configuration, all system properties defined in the parent process are inherited by the forked process.
+In case you want to reset WildFly specific system properties, set this property to an empty value.
```
Comment (spyrkob) [inline docs/guide/intro/index.adoc:135]: Maybe something like `Setting this property to an empty value will reset system properties used by Wildfly server during startup?`

File: docs/guide/intro/index.adoc [hunk 3]
```diff
@@ -13,3 +15,129 @@ implement new WidFly subsystems and expose new server capabilities.
...
+
+|jboss-reset-embedded-system-properties
+|Comma separated list of String
+|NONE
+|When executing a forked process to generate the server configuration, all system properties defined in the parent process are inherited by the forked process.
+In case you want to reset WildFly specific system properties, set this property to an empty value. 
+To erase additional specific system properties, set this property to a comma separated list of system property names to erase.
+
+|jboss-resolved-artifacts-cache
+|Boolean
```
Comment (spyrkob) [inline docs/guide/intro/index.adoc:139]: That should be a String not a Boolean. The value is the path where to generate the cache

Comment (jfdenise) [reply docs/guide/intro/index.adoc]: Thanks, resolved

File: docs/guide/intro/index.adoc (hunk 2 above)
Comment (jfdenise) [reply docs/guide/intro/index.adoc]: Thanks.

File: docs/guide/intro/index.adoc (hunk 1 above)
Comment (jfdenise) [reply docs/guide/intro/index.adoc]: Yes, made it clear.

File: docs/guide/intro/index.adoc [hunk 4]
```diff
@@ -13,3 +15,130 @@ implement new WidFly subsystems and expose new server capabilities.
...
+|'\|' separated list of maven coordinates in the following syntax: GroupId:ArtifactId:Version:[Classifier:]Extension
+|NONE
+|A list of Maven artifact coordinates used to override the artifact versions that are defined in the provisioned WildFly feature-pack.
+For example, to override undertow-core: `io.undertow:undertow-core:2.3.1.Final::jar`
+
+|jboss-reset-embedded-system-properties
+|Comma separated list of String
+|NONE
+|When executing a forked process to generate the server configuration, all system properties defined in the parent process are inherited by the forked process.
+Setting this property to an empty value will reset the system properties used by Wildfly server during startup.
```
Comment (yersan) [inline docs/guide/intro/index.adoc:135]: s/Wildfly/WildFly

File: docs/guide/intro/index.adoc [hunk 5]
```diff
@@ -13,3 +15,130 @@ implement new WidFly subsystems and expose new server capabilities.
...
+To reset WildFly specific system properties and some additional properties, add to the list the names of system property to reset.
```
Comment (yersan) [inline docs/guide/intro/index.adoc:136]: I think it is worth mentioning at the beginning of this sentence that this is a comma-separated list of properties.

File: docs/guide/intro/index.adoc [hunk 6]
```diff
@@ -13,3 +15,130 @@ implement new WidFly subsystems and expose new server capabilities.
...
+<configuration>
+...
+ <plugin-options>
+  <jboss-fork-embedded>true</jboss-fork-embedded>
+ </plugin-options>
+...
+</configuration>
+```
+
+* For the link:https://github.com/wildfly/wildfly-maven-plugin/[Wildfly Maven plugin] configuration: The `<galleon-options>` element contains the options as XML elements.
```
Comment (yersan) [inline docs/guide/intro/index.adoc:48]: s/Wildfly/WildFLy

**Review by yersan — APPROVED** [review]
Just added a very minor comment and one correction, but looks good to me, thanks!

File: docs/guide/intro/index.adoc (hunk 4 above)
Comment (jfdenise) [reply docs/guide/intro/index.adoc]: Thanks

File: docs/guide/intro/index.adoc (hunk 5 above)
Comment (jfdenise) [reply docs/guide/intro/index.adoc]: Done

File: docs/guide/intro/index.adoc (hunk 6 above)
Comment (jfdenise) [reply docs/guide/intro/index.adoc]: Thanks

**Pull request merged**